samples/boards/arc_secure_services/README.rst

```diff
@@ -0,0 +1,126 @@
+  ``arc_mpu_regions.c``. Half of ram and rom is allocated to secure world,
+  the other half is allocated to normal world.
+
+* the memory which is not allocated to secure application is allocated to
+  normal application.
+
+* By default, all the peripheral space is normal mode accessible, i.e.,
+  the peripherals are shared between normal mode and secure mode. If some
+  peripherals are required by secure world, it can be done by adding static
+  mpu entry in ``arc_mpu_regions.c``.
+
+* the interrupts of two internal timers are configure as normal interrupts,
+  so the normal zephyr's kernel tick can work correctly.
+
+* secure interrupts priority > secure threads priority > normal interrupts
+  priority > normal threads priority.
+
+
+Requirements
+************
+
+To use this sample, ARC processor should be equipped with ARC SecureShield.
+In Zephyr, the following board configurations are supported:
+
+* em_starterkit_em7d
+  * secure application: em_starterkit_em7d_secure
+  * normal application: em_starterkit_em7d_normal
+* nsim_sem
+  * secure application: nsim_sem
+  * normal application: nsim_sem_normal
+
+Building and Running
+********************
+
+Building
+========
+
+Secure application
+^^^^^^^^^^^^^^^^^^
+
+First, you should build the secure application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/arc_secure_services
+   :board: em_starterkit_em7d_secure
+   :goals: build debug
+   :compact:
+
+Normal application
+^^^^^^^^^^^^^^^^^^
+
+Currently, in normal application, MPU is not accessible, so no user
+space and mpu-based stack checking.
+
+Here,take :ref:'dining-philosophers-sample' as an example for normal
+application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/philosophers
+   :board: em_starterkit_em7d_normal
+   :goals: build
+   :compact:
+
+Running
+=======
+
+To run this example, there are two ways.
+
+* By bootloader
+
+Bootloader should load secure and normal application into the correct place,
+then jump to the entry of secure application.
+
+* By debugger
+
+Currently, it's recommended to use to gdb debugger to load and run two applications.
+
+For em starter kit, please run the following commands.
+
+.. code-block:: console
+
+    # load secure application first
+    $ cd samples/boards/arc_secure_services
+    $ west debug
+    # load normal application
+    $ monitor load_image <the path of normal application elf file>
+    $ c
+
+For nsim sem, you need two consoles, one for output, one for debug.
+
+In the console for output:
+
+.. code-block:: console
+
+    # open debug server
+    $ cd samples/boards/arc_secure_services
+    $ west debugserver
+
+In the console for debug:
+
+.. code-block:: console
+
+    # open debug server
+    $ cd samples/boards/arc_secure_services
+    $ arc-elf32-gdb zephyr/zephyr.elf
+    $ target remote :3333
+    # load normal application
+    $ load <the path of normal application elf file>
+    # load secure application
+    $ load
```

Comment:
do you need to specify the path of the secure application, or is that on line 120?

Reply:
<the path of normal application elf file> means any zephyr image built in line 73, i will make it more clarified